doc/api/crypto.md

```diff
@@ -4219,8 +4219,12 @@ added:
   input encoding is desired for a string input, user could encode the string
   into a `TypedArray` using either `TextEncoder` or `Buffer.from()` and passing
   the encoded `TypedArray` into this API instead.
-* `outputEncoding` {string|undefined}  [Encoding][encoding] used to encode the
-  returned digest. **Default:** `'hex'`.
+* `options` {Object|string}
```

Comment:
```suggestion
* `options` {Object|string} When it's a string, it is interpreted as `outputEncoding`.
```

Reply:
Yes, i've aligned it with how `{Object|string}` `options` are documented in other places, i.e. in the description.

Reply:
Oh I see it now. Thanks.